installation/scripts/build-kyma-installer.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+echo "The build-kyma-installer.sh script is deprecated and will be removed. Use Kyma CLI instead."
```

Comment:
I wonder if we should delete those warnings, in our case Kyma CLI for this is not an option yet.